From: Sven Roederer <devel-sven@geroedel.de>
Date: Fri, 10 Apr 2020 12:29:58 +0200
Subject: ath79: deactivate PARTITION_ADVANCED

on this crappy hardware it's nearly impossible to add external drives, so
there seems no need for this partition-type.
this will safe some kernel-size

diff --git a/target/linux/ath79/tiny/config-default b/target/linux/ath79/tiny/config-default
index 42243cfc481ab57f0b159535c8a3618c861070f9..6f94e2238953bcfe1c8b70e575ed691f7db71166 100644
--- a/target/linux/ath79/tiny/config-default
+++ b/target/linux/ath79/tiny/config-default
@@ -6,6 +6,7 @@ CONFIG_NET_DSA_MV88E6060=y
 # CONFIG_NET_DSA_TAG_QCA is not set
 CONFIG_NET_DSA_TAG_TRAILER=y
 CONFIG_NET_SWITCHDEV=y
+# CONFIG_PARTITION_ADVANCED is not set
 CONFIG_PHYLINK=y
 CONFIG_PHY_AR7100_USB=y
 CONFIG_PHY_AR7200_USB=y
